q2_types/feature_data/tests/data/taxonomy/blanks-and-comments

```diff
@@ -1,17 +1,17 @@
 
```

Comment:
Why are the comments being removed in this test file? The format itself should still support comments, its just the transformer to Metadata that removes them. The same question applies to the other test files in this PR.

Reply:
Originally, I asked Evan about this particular issue. From that conversation I gathered that since the argument `comment='#'` was removed from the `df = pd.read_csv(filepath, sep='\t', comment='#', skip_blank_lines=True)` line in q2_types/feature_data/_transformer.py, the comments in the test files would need to be removed as well. We may need to speak more about how to support comments while not allowing inline `' #'` insertions to taxonomy data. This causes all data that follows the insertion to be lost, since it is seen as a comment.

Reply:
Thanks @David-Rod - @ebolyen caught me up on your discussion IRL, I am on the same page now. TLDR, the idea is to _stop supporting comments_ in this format, altogether. The fact that the format supported comments was never documented publicly, so we are feeling pretty confident that this change will not pose a problem for historical Artifacts.

Regarding your question about just checking for leading-line comments, I think since we are going to move away from comments in this format you can drop that proposed change. Thanks!

Reply:
Please rename this file to `blanks`.